api/controllers/UserController.js

```diff
@@ -120,5 +120,22 @@ module.exports = _.mapValues({
     const params = req.allParams();
     Validation.requireParams(params, 'name');
     return res.ok(await Validation.usernameAvailable(params.name));
-  }
+  },
+  async forgotUsername (req, res) {
+    const params = req.allParams();
+    Validation.requireParams(params, 'email');
+    const users = await User.find({email: params.email});
+    res.send(202);
```

Comment:
I'm not saying we shouldn't do it.

But "worried about" implies to me that there's a realistic chance it'll happen. "Slightly concerned about" would be better :-P